lib/storage/partition.go

```diff
@@ -62,10 +62,11 @@ var dataFlushInterval = 5 * time.Second
 //
 // This function must be called before initializing the storage.
 func SetDataFlushInterval(d time.Duration) {
-	if d > pendingRowsFlushInterval {
-		dataFlushInterval = d
-		mergeset.SetDataFlushInterval(d)
+	if d < time.Second {
+		logger.Panicf("Invalid value: %s for `-inmemoryDataFlushInterval`,the minimum supported value is 1s", d)
```

Comment:
I don't like panic here, because message refers to the flag name unknown to `lib/storage/partition.go`. In other words, this function doesn't know about `-inmemoryDataFlushInterval` existence, it knows only about `d time.Duration`. 
I think we need just validate the `d` value here and don't accept anything lower than 1s.